main.go

```diff
@@ -88,6 +89,7 @@ func main() {
 		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
 	flag.StringVar(&runnerImage, "runner-image", defaultRunnerImage, "The image name of self-hosted runner container.")
 	flag.StringVar(&dockerImage, "docker-image", defaultDockerImage, "The image name of docker sidecar container.")
+	flag.StringVar(&dockerRegistryMirror, "docker-registry-mirror", "", "The image name of docker sidecar container.")
```

Comment:
This almost looks good to me, but this flag's description seems not good. Could you come up with a better one? 😄

Reply:
😱  how did this happen! will fix this soon. Nice spot.